cmd/create/cluster/cmd.go

```diff
@@ -1917,6 +1943,26 @@ func run(cmd *cobra.Command, _ []string) {
 		}
 	}
 
+	isSharedVPC := false
+	privateHostedZoneID := strings.Trim(args.privateHostedZoneID, " \t")
+	privateHostedZoneRoleARN := strings.Trim(args.privateHostedZoneRoleARN, " \t")
+	baseDomain := strings.Trim(args.baseDomain, " \t")
+	if privateHostedZoneID != "" ||
+		privateHostedZoneRoleARN != "" {
+		if privateHostedZoneID == "" ||
+			privateHostedZoneRoleARN == "" ||
+			baseDomain == "" {
+			r.Reporter.Errorf("To install a cluster into a shared VPC," +
+				"private-hosted-zone-id and private-hosted-zone-role-arn should be provided together")
```

Comment:
The error does not match the check, we are also checking for the `Domain`

Reply:
fixed